app/src/main/java/com/zybooks/pizzaparty/MainActivity.kt

```diff
@@ -35,6 +41,14 @@ class MainActivity : AppCompatActivity() {
             R.id.medium_radio_button -> PizzaCalculator.HungerLevel.MEDIUM
             else -> PizzaCalculator.HungerLevel.RAVENOUS
         }
+        val size = when (whatSize.checkedRadioButtonId){
+            R.id.small_radio_button -> whatSize.checkedRadioButtonId
+            R.id.medium_size_radio_button -> whatSize.checkedRadioButtonId
+            else ->whatSize.checkedRadioButtonId
+
+        }
+
+
         // Get the number of pizzas needed using the Model: Pizza Calculator
         val calculator = PizzaCalculator(numAttend,hungerLevel)
```

Comment:
Edit your code to pass the pizza size information to the Pizza Calculator as a third parameter
` val calculator = PizzaCalculator(numAttend,hungerLevel, size)`